.eslintrc.js

```
@@ -1,47 +1,48 @@
 module.exports = {
-  "globals": {
-    "gettext": true,
-    "ngettext": true,
-    "interpolate": true,
-    "process": true,
-    "require": true
+  'globals': {
+    'gettext': true,
+    'ngettext': true,
+    'interpolate': true,
+    'process': true,
+    'require': true
   },
-  "env": {
-    "browser": true,
-    "es2021": true
+  'env': {
+    'browser': true,
+    'es2021': true,
+    'node': true
```

The node silences errors about "module" and "_dirname" being undefined.